### PR TITLE
fix: use musl and friends

### PR DESCRIPTION
**Pull request by EstebanBorai**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -112,16 +118,16 @@ jobs:
         run: make test_fluvio_install
 
       - name: Install Fluvio CDK
-        run: fluvio install cdk
+        run: fluvio install cdk --develop
```
Comment (nacardin) [inline .github/workflows/ci.yml:121]: Why `--develop` ?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -121,7 +127,7 @@ jobs:
           target: x86_64-unknown-linux-musl
 
       - name: Build Connector
-        run: cdk build -p http-sink
+        run: cdk build -p http-sink --target x86_64-unknown-linux-musl
```
Comment (nacardin) [inline .github/workflows/ci.yml:130]: seems to already be targeting musl, is this needed?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (morenol) [reply .github/workflows/ci.yml:121]: I was just to use cdk latest version. With the new release we can get back to stable version. but maybe it is better to use this version so we can detect earlier issues?

File: .github/workflows/ci.yml (hunk 2 above)
Comment (morenol) [reply .github/workflows/ci.yml:130]: not needed but maybe it is better to be explicit?

**Comment by EstebanBorai** [comment]
May the force be with you @nacardin!

**Pull request closed without merging**